src/nwb_conversion_tools/datainterfaces/ecephys/openephys/openephysdatainterface.py

```diff
@@ -29,25 +31,34 @@ def __init__(
         recording_id: Optional[int] = 0,
         stub_test: Optional[bool] = False,
         verbose: bool = True,
+        spikeextractors_backend: Optional[bool] = False,
```

Comment:
Optional thing again. Oh, also for the `stub_test` above

Reply:
Thanks, done.